ftoolz/unsafe/reflection.py

```diff
@@ -0,0 +1,157 @@
+from typing import List, Set, Type, TypeVar
+
+T = TypeVar('T')
+
+
+def abstract(clz: Type[T]) -> bool:
```

Comment:
I don't think you need `Type[T]`, `Type` should be enough, right? There is no use of the `T` anywhere.